src/garage/tf/algos/dqn.py

```diff
@@ -209,12 +219,20 @@ def train_once(self, itr, paths):
                                mean100ep_qf_loss)
         return last_average_return
 
-    def get_itr_snapshot(self, itr):
-        """Get snapshot of the policy."""
-        return dict(itr=itr, policy=self.policy)
+    def optimize_policy(self, itr, samples_data):
+        """Optimize network using experiences from replay buffer.
+
+        Args:
+            itr (int): Iterations.
+            samples_data (list): Processed batch data.
+
+        Returns:
+            numpy.float64: Loss of policy.
+
+        """
+        del itr
```

Comment:
This seems strange to me. If this is not needed inside the function, shouldn't it not be passed, or at least marked as `_`?

Reply:
This is to fight with the recent change on stricter pylint checks. Pylint detects and rejects unused variables, and this `del` line will suppress this check.
Since `train_once()` is a virtial method, remove `itr` will require removing it from the parent class, so is not an option. 
If marked with `_`, pylint will complain about the name does not match with that in the parent class.